testing/integration-tests/universal-profiling-test/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+import java.nio.file.Paths
+
+plugins {
+  id("elastic-otel.test-with-agent-conventions")
+}
+
+dependencies {
+  testImplementation(project(":testing-common"))
+}
+
+
+tasks.withType<Test>() {
+
+  // We need a short temporary path, because the socket file path length is limited to only ~100 chars
```

Comment:
[question] do you have more details where this limitation comes from ? If there is any documentation it could be worth adding a link for future reference.

Reply:
It's a unix limitation:
https://unix.stackexchange.com/questions/367008/why-is-socket-path-length-limited-to-a-hundred-chars